pkg/term/tc_other.go

```diff
@@ -1,21 +1,21 @@
 // +build !windows
-// +build !linux !cgo
-// +build !linux !ppc64le
```

Comment:
1.) This should still catch those, because they are !(windows) and (!solaris or !cgo).

2.) Sort of, it's defining a func named tcget / tcsets as the unix syscall tcget/tcset, where before it was defining them as the C ones. I think it should stay as other because it's building either the solaris file or this (other) one. If it was named tc.go and tc_solaris.go I don't think that would be as obvious, and the build constraints in these files are already confusing enough.